.github/workflows/build_cmake.yml

```diff
@@ -54,11 +54,8 @@ jobs:
       with:
         python-version: '3.x'
 
-    - name: Install build dependencies
-      run: python -m pip install -r python_build_modules.txt
-
     - name: Update Submodules
-      run: git submodule update --init --recursive
+      run: git submodule update --init --recursive --depth=1
```

Comment:
Note: this is a bonus change because I got tired of waiting so long for each successive build failure. Doesn't improve overall build time by much, but time-to-early-failure is a nice improvement.